src/frequenz/sdk/actor/_channel_registry.py

```diff
@@ -0,0 +1,150 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
```

Comment:
2024 :laughing:

Reply:
It was created in 2022 :)

My research shows that copyright notices, and in particular years beyond the year of creation are not that necessary. I did my research a while ago using the good ol' manual Google search, but now checking with ChatGPT to see what they think about it:

Q: Is it necessary to add the copyright year to copyright notices in source code? And what about  keeping it up to date every time the file is updated?

> Including the copyright year in source code copyright notices is customary but not strictly necessary. The year often indicates the first year of publication of the work. Keeping it updated annually is a common practice, though not legally required. The main purpose of the year in a copyright notice is to give an indication of the earliest date of copyright.
> 
> Once a work is copyrighted, it remains so regardless of whether the year in the notice is updated. However, updating the year can be beneficial for demonstrating the timeline of development and updates to the work. This practice can be particularly relevant in open-source projects, where it can help track changes and the evolution of the code over time.
> 
> In summary, while not legally mandatory, adding and updating the year in copyright notices is a good practice for clarity and documentation purposes.

When using a VCS, tracking the date of changes is trivial if it would become necessary for any legal reason.

Reply:
> It was created in 2022 :)

I think the complete removing & adding confused me here then